sys/kern/vmem.c

```diff
@@ -251,56 +238,41 @@ int vmem_alloc(vmem_t *vm, vmem_size_t size, vmem_addr_t *addrp) {
   vmem_check_sanity(vm);
 
   bt_t *bt;
-  vmem_addr_t start;
   bool found = false;
   for (vmem_freelist_t *list = first; list < end; list++) {
     bt = LIST_FIRST(list);
-    if (bt != NULL && vmem_align_and_fit(bt, size, align, &start) == 0) {
+    if (bt != NULL) {
+      /* This is INSTANTFIT strategy, we know that any segment found on these
```

Comment:
Ditto?

Reply:
I'd leave a mention that this is instant-fit... But I've changed caps to regular font: https://github.com/cahirwpz/mimiker/pull/593/commits/1c0c2fff9b1a44e87ef5a3a2f11e3fb6be1b147f